addons/web/static/src/views/fields/properties/property_definition_selection.js

```diff
@@ -198,6 +229,25 @@ export class PropertyDefinitionSelection extends Component {
         options.splice(optionIndex, 1);
         this.props.onOptionsChange(options);
     }
+
+    /**
+     * Move an option after an other one.
+     *
+     * @param {string} from, the option to move
+     * @param {string} to, the target option
+     *      (null if we move the option at the first index)
+     */
+    onOptionMoveTo(from, to) {
+        const options = this.optionsVisible;
+        const fromIndex = options.findIndex((option) => option[0] == from);
+        let toIndex = options.findIndex((option) => option[0] == to);
+        if (toIndex < fromIndex) {
+            // the first splice operation will change the index
+            toIndex++;
+        }
+        options.splice(toIndex, 0, options.splice(fromIndex, 1)[0]);
+        this.props.onOptionsChange(options);
```

Comment:
Shouldn't we handle the case where we are moving an option that we edited but did not "blur" yet (thus for which the changes were not propagated) ?
For the moment it creates a duplicate key if you move a new option with a value, or it rollbacks to the previous value if you move an existing property that you edited.
Maybe by forcing a blur in `onWillStartDrag`? (not sure if it will work, could maybe mess with the draggable hook)
This could maybe also allow to get rid of the `element.blur()` in the `enable` handler (it seems strange to me to actually do something in it, imo it should only indicate if the Sortable hook is enabled or not, and not depend on the element we try to move).